app/services/idv/proofer.rb

```diff
@@ -0,0 +1,72 @@
+module Idv
+  module Proofer
+    ATTRIBUTES = %i[
+      uuid
+      first_name last_name middle_name gen
+      address1 address2 city state zipcode
+      prev_address1 prev_address2 prev_city prev_state prev_zipcode
+      ssn dob phone email
+      ccn mortgage home_equity_line auto_loan
+      bank_account bank_account_type bank_routing
+      state_id_number state_id_type state_id_jurisdiction
+    ].freeze
+
+    STAGES = %i[resolution state_id address].freeze
+
+    @vendors = {}
+
+    class << self
+      attr_accessor :configuration
+
+      def configure
+        self.configuration ||= Configuration.new
+        yield(configuration)
+      end
+
+      class Configuration
+        attr_accessor :mock_fallback, :raise_on_missing_proofers
+        def initialize
+          @mock_fallback = true
+          @raise_on_missing_proofers = true
+        end
+      end
+
+      def attribute?(key)
+        ATTRIBUTES.include?(key&.to_sym)
+      end
+
+      def get_vendor(stage)
+        @vendors[stage]
+      end
+
+      def load_vendors!
+        external_vendors = ::Proofer::Base.subclasses
```

Comment:
This is all or nothing with proofing vendors right? In other words, we can't use just AAMVA and fallback to mock for other stages if we want to test the AAMVA gem in QA?